src/main.rs

```diff
@@ -13,6 +13,13 @@ use simple_logger::SimpleLogger;
 fn main() {
     SimpleLogger::new().init().unwrap();
 
+    let _sentry_dns = "https://891eb4b6f8ff4f959fd76a587d9ab302@o4505481987489792.ingest.sentry.io/4505482139140096";
```

Comment:
Fuck. Guys.

```suggestion
    let _sentry_dsn = "https://891eb4b6f8ff4f959fd76a587d9ab302@o4505481987489792.ingest.sentry.io/4505482139140096";
```

Reply:
@renatgalimov I'm sorry, I've misread DSN as "DNS" first time I've looked into Sentry documentation and it stayed that way. I'll remember for the future.